miraheze/mediawiki/mwimport.py

```
@@ -0,0 +1,146 @@
+    version = args.version or get_version(args.wiki)
+
+    scripts = get_scripts(args)
+    scripts = [
+        ["sudo", "-u", "www-data", "php", f"/srv/mediawiki/{version}/maintenance/run.php", f"--wiki={args.wiki}", *script] for script in scripts
+    ]
+
+    print("Will run:")
+    for script in scripts:
+        print(f"* {shlex.join(script)}")
+    if not args.confirm:
+        if input("Type 'Y' to confirm: ").upper() != "Y":
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)